Source/Models/Application/DataTransfer/MovieDTO.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace TDP.Models.Application;
+
+public record MovieDTO
```

Comment:
suggestion: do not suffix these with DTO, keep the simple name instead and drop them under DataTransfer namespace in order to explicitly differentiate them from domain entitites.